doc/src/explore/move-examples/basics.md

```diff
@@ -24,6 +22,99 @@ Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-fram
 examples = "0x0"
 ```
 
+The manifest itself contains a number of sections, primary of which are:
+
+- `[package]` - includes package metadata such as name and author
+- `[dependencies]` - specifies dependencies of the project
+- `[addresses]` - address aliases (e.g., `@me` will be treated as a `0x0` address)
+
+Dependencies in the `[dependencies]` section are in the form:
+
+```
+<string> = { local = <string>, addr_subst* = { (<string> = (<string> | "<hex_address>"))+ } } # local dependencies
+<string> = { git = <URL ending in .git>, subdir = <path to dir containing Move.toml inside git repo>, rev = <git commit hash>, addr_subst* = { (<string> = (<string> | "<hex_address>"))+ } } # git dependencies
```

Comment:
The `git` field here does not have to always contain a URL that ends in `.git`.  It can contain and URL, or filepath that points to a git repo (i.e. you should be able to call `git clone` with this field).

Reply:
GTK. Will update.